x-pack/plugins/apm/server/lib/index_pattern/index.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { Server } from 'hapi';
+// @ts-ignore
+import { getSavedObjects } from '../../../../../../src/legacy/core_plugins/kibana/server/tutorials/apm/saved_objects/get_saved_objects';
```

Comment:
I need to find a better way to share this code between the plugin and core, because this fails production build.

Reply:
perhaps it belongs in a linked package at `packages/kbn-plugin-apm` or something?

Reply:
I think you'll need to talk to the platform team about this. This is a good example of a shared dependency that we want to import without too much hassle.

Reply:
This should be fine after we merge: https://github.com/elastic/kibana/pull/32722

Reply:
relative imports across x-pack and core should be possible soon once https://github.com/elastic/kibana/pull/32722 merges, so i'll hold off until then.